src/pages/signup/Signup.tsx

```diff
@@ -72,7 +72,7 @@ const Signup = () => {
               setIsPasswordValid={setIsPasswordValid}
             />
           </Box>
-          <ButtonPrimary loading={isPending} disabled={isPasswordValid}>
+          <ButtonPrimary loading={isPending} disabled={!isPasswordValid}>
```

Comment:
you can remove this, I updated this in the forgot password branch, and I might give a merge conflict.

Reply:
if changes are the same, I don't think it'll occur a conflict. Or it's different?

Reply:
it should be disabled={!isPasswordValid} because if password is Not valid button should be disabled